src/powershell-native/nativemsh/pwrshplugin/pwrshplugin.h

```diff
@@ -490,6 +490,11 @@ class PwrshPlugInMediator
         {
             iMajorVersion = iVPSMajorVersion;
 
+            if (NULL != wszMgdPlugInFileName)
```

Comment:
This already occurs on line 671. It seems unnecessary to do it twice. Did the scanner require it?

Reply:
This check is added because wszMgdPlugInFileName was not being set in all paths; specifically error paths.